pyproject.toml

```diff
@@ -1,3 +1,85 @@
+#------------------------------------------------------------------------------
+# Copyright (c) The pyfar developers
+#------------------------------------------------------------------------------
+
+[project]
+name = "pyfar"
+version = "0.6.10"
+description = "Project for data formats in acoustics."
+readme = "README.md"
+license = {file = "LICENSE"}
+requires-python = ">=3.8"
```

Comment:
We have to change this manually to 3.9 after releasing 0.7.0

Reply:
already done, because it's also already included in the circleci config